### DISCUSSION TITLE
Option for marker timestamp to be IRL time

### DISCUSSION THREAD

**Discussion by Daniel-H123** [acemod/ACE3#9119] [Ideas]
When I am use ACE marker timestamps I'm always bothered by the fact that it uses the in game time. Most of the time I don't even care what time it is in game and I also don't know how fast time excelleration is. Wouldn't it be much easier to just add an option in the addon options to use the systemtime? I know it would be different for players in other timezones, but mostly I am playing with friends in the same timezone. I would like to know how other people think about this.

**Comment by PabstMirror** [comment]
can do via https://github.com/acemod/ACE3/commit/15372558b2daf32178e369a27d467e7b71a52543

Like you said because of timezones this won't work for most people in multiplayer
but I don't see any harm in adding it as a setting

**Reply by Daniel-H123** [reply]
Thx for your response

Edited your commit #9121 9b5ee1bf7041bd72eec93cd9e2bd636b9efa41fd

I think this should be better. It keeps the option for users to use the 12hrs formatting. 

It also adds the option to select a custom UTC time zone. After I posted the question I thought it can be likely that some people don't have control over the server's time zone.

**Comment by Daniel-H123** [comment]
Merged https://github.com/acemod/ACE3/pull/9121
